src/wp-includes/block-template.php

```diff
@@ -268,9 +268,15 @@ function get_the_block_template_html() {
 	$content = wp_filter_content_tags( $content, 'template' );
 	$content = str_replace( ']]>', ']]&gt;', $content );
 
+	/* 
+	 * Ticket : 61956
+	 * Filter to allow modification of the wrapper element.
+	 */
+	$wrapper_element = apply_filters( 'wp_site_blocks_wrapper_element', 'div', $content );
```

Comment:
Add filter doc. per https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters.

Reply:
@mukeshpanchal27 Thanks for reviewing this, I have implemented the suggestions please check.